component/common/src/main/java/org/wso2/carbon/identity/smsotp/common/util/Utils.java

```diff
@@ -128,6 +128,14 @@ private static void sanitizeAndPopulateConfigs(Properties properties) throws SMS
                 Integer.parseInt(otpResendThrottleIntervalValue) * 1000 : Constants.DEFAULT_RESEND_THROTTLE_INTERVAL;
         configs.setResendThrottleInterval(resendThrottleInterval);
 
+        //Maximum allowed validation attempts defaults to 5 if its not specified as a property in deployment.toml file.
```

Comment:
Shall we add a space after `//`?

Reply:
Resolved with 69b280575c654b56917dae1fdf53ea71618ef29a